template_workflows/project_automation/stale.yml

```
@@ -6,6 +6,10 @@ on:
 jobs:
   stale:
     runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
```

```suggestion
    permissions:
      actions: write
      issues: write
      pull-requests: write

```
In https://github.com/actions/stale?tab=readme-ov-file#recommended-permissions they specify the `actions` permission as well, though it's not immediately clear to me what it's used for